doc/how_to_build_win.md

```diff
@@ -1,47 +1,44 @@
+This software can be built using Visual Studio 2015 or above and Qt 5.9(later versions still not working correctly.)
 
 ## Required Software
 
-### Visual Studio Express 2015 for Windows Desktop
+### Visual Studio Express 2015 or higher for Windows Desktop
 - https://www.visualstudio.com/vs/older-downloads/
-- Make sure that the version is "2015" and that the target platform is "for Windows Desktop", not "for Windows".
-- Community and Professional versions of Visual Studio 2015 for Windows Desktop also work.
-- Visual Studio 2017 is currently unsupported.
+- Make sure the target platform is "for Windows Desktop", not "for Windows".
+- Community and Professional versions of Visual Studio for Windows Desktop also work.
 - During the installation, make sure to select all the Visual C++ packages.
 
 ### CMake
 - https://cmake.org/download/
 - This will be used to create the `MSVC 2015` project file.
 
-## Acquiring the Source Code
-- Clone the base repository.
-- Throughout the explanation `$opentoonz` will represent the root for the base repository.
-- Visual Studio cannot recognize UTF-8 without BOM source code properly.  Furthermore, since the endline character is represented with only the LF character, one line comments in Japanese will often cause the following line to be treated as a comment by `MSVS` as well.
-- In order to prevent this, please change the following setting in git so that it will preserve the proper endline characters:
-  - `git config core.safecrlf true`
-
-## Installation of Required Libraries
-Because of the size of these libraries, they are not maintained in the git repository.
-They will have to be installed seperately as follows.
-
-### `lib` and `dll`
-- `lib` and `dll` files are tracked by [Git Large File Storage](https://git-lfs.github.com/).
-- Execute `git lfs pull` after `git clone` by using the `lfs` client.
-
 ### Qt
 - https://www.qt.io/download-open-source/
 - Qt is a cross-platform GUI framework.
-- Install Qt 5.9 (64-bit version, tested up to 5.9.4) with the [Qt Online Installer for Windows](http://download.qt.io/official_releases/online_installers/qt-unified-windows-x86-online.exe).
+- Install Qt 5.9 (64-bit version, tested up to 5.9.9) with the [Qt Online Installer for Windows](http://download.qt.io/official_releases/online_installers/qt-unified-windows-x86-online.exe).
 
 ### boost
 - Boost 1.55.0 or later is required (tested up to 1.61.0).
 - http://www.boost.org/users/history/version_1_61_0.html
-- Download boost_1_61_0.zip from the above link.  Extract all contents to the '$opentoonz/thirdparty/boost' directory.
+- Download boost_1_73_0.zip from the above link.  Extract all contents to the '$opentoonz/thirdparty/boost' directory.
 - Install the following patch (Make the changes listed in the patch file), if you use Boost 1.55.0 with Visual Studio 2013.
   - https://svn.boost.org/trac/boost/attachment/ticket/9369/vc12_fix_has_member_function_callable_with.patch
 
+## Acquiring the Source Code
```

Comment:
This whole section should be where it originally was.  You can't place the boost directory in the right folder unless you have the source code.

Reply:
Okay I will fix.  Do I just edit it and resend it?  Or should I just close it?